src/source/_static/custom.css

```diff
@@ -12,3 +12,14 @@
 .strike {
     text-decoration: line-through;
 }
+
+.larger-collapse {
+    font-size: 120%;
+    font-weight: bold;
+    padding-bottom: 6px;
+}
+
+.larger-collapse p {
+    font-size: 100%;
+    font-weight: normal;
+}
```

Comment:
where does this come from? We typically don't modify CSS by hand, and you said you didn't re-generate

Reply:
That was added by me. I did not like formatting of Questions and Answers section. But it can be removed we do not accept manual css modifications.

Reply:
I'm fine having this, as long as it doesn't get overwritten during generation again :D

Reply:
It seems custom.css is not overwritten